zebra/zebra_srv6_vty.c

```diff
@@ -1408,9 +1408,8 @@ static int zebra_sr_config(struct vty *vty)
 			if (locator->node_bits_length)
 				vty_out(vty, " node-len %u",
 					locator->node_bits_length);
-			if (locator->function_bits_length)
```

Comment:
Shouldn't we just do with another check `&& locator->function_bits_length != ZEBRA_SRV6_FUNCTION_LENGTH)` (which is a default, right)?